nixos/tests/nixseparatedebuginfod.nix

```diff
@@ -0,0 +1,80 @@
+import ./make-test-python.nix ({ pkgs, lib, ... }:
+let
+  secret-key = "key-name:/COlMSRbehSh6YSruJWjL+R0JXQUKuPEn96fIb+pLokEJUjcK/2Gv8Ai96D7JGay5gDeUTx5wdpPgNvum9YtwA==";
```

Comment:
I think we already have some snake oil signing keys which should be re-used but I didn't check eight now.

Reply:
I could only find snake oil x509 certs. For just two strings, that have no expiration date, the current state does not seem to bad.

Reply:
We have some in https://github.com/NixOS/nixpkgs/blob/8f3c9dec12d8eafaa09de23e7f737d95eaf640ff/nixos/tests/harmonia.nix we could share but whatever. Those are easy to generate and don't expire.